platformio.ini

```diff
@@ -8,29 +8,23 @@
 ; http://docs.platformio.org/en/stable/projectconf.html
 
 [platformio]
-default_envs = openevse
+#default_envs = openevse
 src_dir = firmware/open_evse
 
 [common]
 lib_deps =
 upload_protocol = usbasp
-upload_flags = -e
-
-[env:openevse]
-
-# Arduino 1.6.14
-#platform = atmelavr@1.4.0
-# Arduino 1.6.17
-#platform = atmelavr@1.4.1
-# Arduino 1.6.23
-platform = atmelavr@1.15.0
-
-board = openevse
-framework = arduino
-lib_deps = ${common.lib_deps}
-upload_protocol = ${common.upload_protocol}
-upload_flags = ${common.upload_flags}
+upload_flags = "-e -B0.5"
+version = 7.1.5
+eu_build_flags =
+  -D PP_AUTO_AMPACITY
```

Comment:
`PP_AUTO_AMPACITY` only applies to `emonevse` and `emonevse_3ph` environments, NOT `openevse_eu`

Reply:
`PP_AUTO_AMPACITY` should be removed from common `eu_build_flags` and only applied to `emonevse` and `emonevse_3ph` environments. Sorry, I can't see how I can make this change myself.